modules/030-cloud-provider-yandex/templates/cloud-metrics-exporter/pod-monitor.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if and .Values.cloudProviderYandex.internal.providerDiscoveryData.natInstanceName .Values.cloudProviderYandex.internal.providerDiscoveryData.monitoringAPIKey }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: yandex-nat-instance-metrics
+  namespace: d8-monitoring
+  {{- include "helm_lib_module_labels" (list . (dict "prometheus" "main" "app" "cloud-metrics-exporter")) | nindent 2 }}
+spec:
+  jobLabel: app
+  podMetricsEndpoints:
+    - port: https-metrics
+      path: /metrics/compute
+      scheme: https
+      bearerTokenSecret:
+        name: "prometheus-token"
+        key: "token"
+      tlsConfig:
+        insecureSkipVerify: true
+      honorLabels: true
+      scrapeTimeout: "60s"
```

Comment:
```suggestion
```
Scrape timeout cannot exceed the scrape interval, which equals the 30s by default. You need to specify a value less than it, e.g., 25s (but hardcode is also no good here).